tests/local/cheyenne/model_test.sh

```diff
@@ -176,6 +177,7 @@ module list
 # Python Env
 deactivate > /dev/null 2>&1
 source /glade/p/cisl/nwc/model_testing_env/wrf_hydro_nwm_test/bin/activate || exit 9
```

Comment:
@rcabell

Reply:
Strange, I reloaded it this morning. Looks good now - thanks!